packages/react/src/app-layout/AppLayoutSidebarDialog.tsx

```diff
@@ -37,48 +43,12 @@ export function AppLayoutSidebarDialog({
 		return () => window.removeEventListener('keydown', handleKeyDown);
 	}, [closeMobileMenu]);
 
-	// The following `useEffect` will add `aria-hidden="true"` to every direct child of the `body` element when the modal is opened.
-	// This is because `aria-modal` is not yet supported by all browsers (https://a11ysupport.io/tech/aria/aria-modal_attribute).
-	// This fixes a bug in certain devices where focus is not trapped correctly such as VoiceOver on iOS.
-	// This has been inspired by Reach UI (https://github.com/reach/reach-ui/blob/main/packages/dialog/src/index.tsx)
-	useEffect(() => {
-		if (!isMobileMenuOpen || !dialogRef.current) return;
-		const rootNodes: Element[] = [];
-		const originalAttrs: (string | null)[] = [];
-		document.querySelectorAll('body > *').forEach(function (node) {
-			if (node === dialogRef.current) return;
-			const attr = node.getAttribute('aria-hidden');
-			const alreadyHidden = attr !== null && attr !== 'false';
-			if (alreadyHidden) return;
-			rootNodes.push(node);
-			originalAttrs.push(attr);
-			node.setAttribute('aria-hidden', 'true');
-		});
-		return () => {
-			rootNodes.forEach((node, index) => {
-				const originalValue = originalAttrs[index];
-				if (originalValue === null) {
-					node.removeAttribute('aria-hidden');
-				} else {
-					node.setAttribute('aria-hidden', originalValue);
-				}
-			});
-		};
-	}, [isMobileMenuOpen]);
-
 	// Animation styles
 	const prefersReducedMotion = usePrefersReducedMotion();
-	const [overlayTransitions] = useTransition(isMobileMenuOpen, () => ({
-		from: { opacity: 0 },
-		enter: { opacity: 1 },
-		leave: { opacity: 0 },
-		config: { duration: 150 },
-		immediate: prefersReducedMotion,
-	}));
 	const [dialogTransitions] = useTransition(isMobileMenuOpen, () => ({
```

Comment:
I just learnt this which is pretty cool, I was able to combine these 2 transitions into 1.